packages/rancher-wins-upgrader/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/wins/releases/download/v0.1.0/dist.rancher-wins-upgrader-0.0.1.tgz
```

Comment:
0.0.1 for the upgrader?

Reply:
Yes, v0.1.0 is the wins release and v0.0.1 is the Helm chart version https://github.com/rancher/wins/blob/b1e2d886d3e86642e3abfe40b2d127129325ceb8/charts/rancher-wins-upgrader/Chart.yaml#L3